### ISSUE TITLE
Front-End

### ISSUE THREAD

**Issue by KelsonF** [hydralauncher/hydra-website#3]
I'm a Brazilian front-end developer (at least I'm trying) and i would like to know if I can improve the Hydra website, i would love to help you guys with the project and learn more to help in the main project (the launcher)

**Comment by hydralauncher** [comment]
You can join our [discord](https://discord.gg/hydralauncher) and help the collaborator who is updating the website in this pr https://github.com/hydralauncher/hydra-website/pull/2

you will need to go to the roles channel and get the 'colaborador' role to access the dev channel

**Comment by GabrielFinotti** [comment]
Good evening, I've been following the development of the project for a while, I have more or less 2 years of experience in frontend development and 6 months with backend. I would like to know if I can collaborate with the project to develop the website, and in the future, if possible, also with the main launcher.

**Comment by hydralauncher** [comment]
@GabrielFinotti of course! This is a project kept by the community. Feel free to join our Discord likewise.

**Comment by GabrielFinotti** [comment]
@hydralauncher thanks bro, how do I become a contributor?

**Comment by hydralauncher** [comment]
@GabrielFinotti just react to the message on the roles channel, and then the dev chats will be unlocked.

**Comment by GabrielFinotti** [comment]
@hydralauncher thanks very much bro!!
